picard/ui/coverartbox.py

```diff
@@ -136,6 +136,7 @@ def dragMoveEvent(event):
         event.acceptProposedAction()
 
     def dropEvent(self, event):
+        event.setDropAction(QtCore.Qt.DropAction.CopyAction)
```

Comment:
Does this work without breaking normal move actions inside Picard? My idea would have been to only do this if the drop originates from outside.

Reply:
As I commented previously elsewhere, one `proposedAction` is do nothing or Ignore. IMO you should NOT change this to a CopyAction.

Reply:
Picard only accepts copy and move actions. There is nothing else at that part of code.